pants.ini

```diff
@@ -450,3 +450,9 @@ enable_libc_search: True
 
 [sourcefile-validation]
 config: @build-support/regexes/config.yaml
+
+[cache.resolve.coursier]
+# Only use local artifact for coursier because the cache content contains abs path
```

Comment:
Does this mean that every consumer of coursier will need to do this, in order to not experience portability bugs?

Reply:
This is not required, but makes it more performance. 

The task still validate the paths after hitting the cache, so it will recognize the cache isn't usable then call coursier again.

Reply:
It would be good to loop back and put that in a comment probably.

Reply:
Added. Thanks!